src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -616,6 +626,16 @@
         </state>
     </channel-type>
 
+    <!-- Energy - Power Consumption -->
+    <channel-type id="meter_kvar">
+        <item-type>Number</item-type>
+        <label>Electric Power Consumption</label>
```

Comment:
Suggest to use consistent naming here - you added "Energy Consumption" above but "Electric Power Consumption" here. Either use "Electric" in both or neither.

Reply:
You are right again. As the "Electric" is not only in "Electric Power Consumption", but also in "Electric Current", it should be "Electric Energy Consumption". Sorry for that.